scripts/stack_helpers.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+is_stack_created() {
+    local files=0
+    files=$(find ~/.elastic-package -type f -name "docker-compose.yml" | wc -l)
+    if [ "${files}" -gt 0 ]; then
+        return 0
+    fi
+    return 1
+}
+
```

Comment:
Helper function to know if `elastic-package stack up` command has been performed or not.

There could be packages that fail in `elastic-package check` command and if so, the Elastic stack has not been created.

This is helpful to avoid running commands like `elastic-package stack <dump|down>` that are going to fail.